### PR TITLE
feature/invgen-40309: add a link to the about dialog

### PR DESCRIPTION
**Pull request by msimko81**
A new link referencing README.md is added to the about dialog.

### CODE REVIEW

**Review by slavikz-centrum — APPROVED** [review]

File: howto.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
-#TODO: describe what application can do for customers and how to log-in to the application (to be able to upload projects)
+# Autodesk Configuration with Design Automation for Inventor
+Using the Autodesk Configuration with Design Automation for Inventor web application one can easily manage and edit Inventor projects using only web browser and internet connection. The following operations are available:
```
Comment (andersco) [inline howto.md:2]:
```suggestion
Using the Autodesk Configuration with Design Automation for Inventor web application one can easily manage and edit Inventor projects using only a web browser and an internet connection. The following operations are available:
```

File: howto.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+- manage custom Inventor projects (list, upload, delete)
+- interactively display the model
+- change editable parameters and update the model
+- list bill of material
```
Comment (andersco) [inline howto.md:6]:
```suggestion
- list bill of materials
```

File: howto.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+- generate model drawing
+- download individual Inventor project assemblies (.IAM), Revit 3D model (.RFA), bill of material (.CSV) and model drawing (.PDF)
 
-Part of this file shall be a link explaining how the price is calculated: https://forge.autodesk.com/pricing
+Application is available using this link: https://inventor-config-demo.autodesk.io/ (TODO: change to the release link)
```
Comment (andersco) [inline howto.md:10]:
```suggestion
The application is available using this link: https://inventor-config-demo.autodesk.io
```

File: howto.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
 
-https://jira.autodesk.com/browse/INVGEN-40309
+Non authenticated anonymous user can work with two predefined projects: Wheel and Wrench.
```
Comment (andersco) [inline howto.md:12]:
```suggestion
Non authenticated anonymous users can work with two predefined projects: Wheel and Wrench. 
```

File: howto.md [hunk 5]
```diff
@@ -1,5 +1,16 @@
...
+- generate model drawing
+- download individual Inventor project assemblies (.IAM), Revit 3D model (.RFA), bill of material (.CSV) and model drawing (.PDF)
 
-Part of this file shall be a link explaining how the price is calculated: https://forge.autodesk.com/pricing
+Application is available using this link: https://inventor-config-demo.autodesk.io/ (TODO: change to the release link)
 
-https://jira.autodesk.com/browse/INVGEN-40309
+Non authenticated anonymous user can work with two predefined projects: Wheel and Wrench. 
+
+To be able to upload custom projects, one needs to authenticate using his Autodesk Forge account. To sign up for the Forge account, follow instructions in this web page: https://forge.autodesk.com/.
```
Comment (andersco) [inline howto.md:14]:
```suggestion
To be able to upload custom projects, one needs to authenticate using their Autodesk Forge account. To sign up for the Forge account, follow the instructions here: https://forge.autodesk.com/.
```

File: howto.md [hunk 6]
```diff
@@ -1,5 +1,16 @@
...
+
+Every operation (i.e. model update after change of parameters, drawing creation) is computed on a cloud. For this purpose, Cloud Credits are required. Free Cloud Credits are provided to try Forge after sign up. For more information about pricing, please see https://forge.autodesk.com/pricing.
```
Comment (andersco) [inline howto.md:16]:
```suggestion
Every operation (i.e. model update after change of parameters, drawing creation) is computed in the cloud. For this purpose, Cloud Credits are required. Free Cloud Credits are provided to try Forge after signing up. For more information about pricing, please see https://forge.autodesk.com/pricing.
```

**Review by erymski — APPROVED** [review]
I guess `DO NOT MERGE` label can be removed?

**Pull request merged**